samples/dashboards/demo/live-weather/demo.js

```diff
@@ -0,0 +1,917 @@
+                cell: 'kpi-humidity',
+                type: 'KPI',
+                columnName: 'humidity',
+                chartOptions: {
+                    ...KPIChartOptions,
+                    title: {
+                        text: paramConfig.getColumnHeader('humidity'),
+                        verticalAlign: 'bottom',
+                        widthAdjust: 0
+                    },
+                    yAxis: {
+                        accessibility: {
+                            description: paramConfig.getColumnHeader('humidity')
+                        },
+                        max: paramConfig.humidity.max,
+                        min: paramConfig.humidity.min
+                    }
+                },
+                events: {
+                    click: function () {
+                        // Update board
+                        activeParam = paramConfig.humidity;
+                        updateBoard(board, activeCity, 'humidity');
+                    }
+                },
+                states: {
+                    active: {
+                        enabled: true
+                    },
+                    hover: {
+                        enabled: true
+                    }
+                }
+            }, {
+                cell: 'selection-grid',
+                type: 'DataGrid',
+                sync: {
+                    highlight: true
+                },
+                dataGridOptions: {
+                    cellHeight: 38,
+                    editable: false,
+                    columns: {
+                        time: {
+                            headerFormat: 'Time UTC',
+                            cellFormatter: function () {
+                                return Highcharts.dateFormat('%d/%m, %H:%M', this.value);
+                            }
+                        },
+                        temperature: {
+                            headerFormat: paramConfig.getColumnHeader('temperature'),
+                            cellFormat: '{value:.1f}'
+                        },
+                        pressure: {
+                            headerFormat: paramConfig.getColumnHeader('pressure'),
+                            cellFormat: '{value:.1f}'
+                        },
+                        humidity: {
+                            headerFormat: paramConfig.getColumnHeader('humidity'),
+                            cellFormat: '{value:.1f}'
+                        }
+                    }
+                }
+            }, {
+                cell: 'city-chart',
+                type: 'Highcharts',
+                columnAssignment: {
+                    time: 'x',
+                    temperature: 'y'
+                },
+                sync: {
+                    highlight: true
+                },
+                chartOptions: {
+                    chart: {
+                        spacing: [40, 40, 40, 10],
+                        styledMode: true,
```

Comment:
styledMode is enabled by default in all `HighchartsComponents`

Reply:
Fixed.